packages/frontend/apps/ios/App/App/AffineHttpHandler.swift

```diff
@@ -0,0 +1,114 @@
+//
```

Comment:
split into a standalone pr

Reply:
Due to the xcodeproj, it is complicated to do this